services/cii-best-practices/cii-best-practices.service.js

```diff
@@ -0,0 +1,162 @@
+'use strict'
+
+const Joi = require('joi')
+const BaseJsonService = require('../base-json')
+const {
+  colorScale,
+  coveragePercentage: coveragePercentageColor,
+} = require('../../lib/color-formatters')
+
+const ciiBestPracticesSchema = Joi.object({
+  badge_level: Joi.string().required(),
+  tiered_percentage: Joi.number().required(),
+}).required()
+
+const keywords = ['cii', 'cii best practices', 'core infrastructure initiative']
+
+module.exports = class CIIBestPracticesService extends BaseJsonService {
+  static render({ label, message, color }) {
+    return {
+      label,
+      message,
+      color,
+    }
+  }
+
+  static get defaultBadgeData() {
+    return { color: 'blue', label: 'cii' }
+  }
+
+  static get category() {
+    return 'other'
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'CII Best Practices Level',
+        namedParams: {
+          metric: 'level',
+          projectId: '1',
+        },
+        pattern: 'level/:projectId',
+        staticExample: this.render({
+          label: 'cii',
+          message: 'gold',
+          color: '#E9C504',
+        }),
+        keywords,
+      },
+      {
+        title: 'CII Best Practices Tiered Percentage',
+        namedParams: {
+          metric: 'percentage',
+          projectId: '29',
+        },
+        pattern: 'percentage/:projectId',
+        staticExample: this.render({
+          label: 'cii',
+          message: '107%',
+          color: 'brightgreen',
+        }),
+        keywords,
+      },
+      {
+        title: 'CII Best Practices Default',
```

Comment:
Maybe we could make the meaning of "default" clearer by adding some documentation to the example? Pretty much taken from your PR's description: "This badge uses the same message and color scale as the native CII one, but with all the configuration and goodness that Shields provides!". You can refer to our [Matrix](https://github.com/badges/shields/blob/master/services/matrix/matrix.service.js) badge for an example of how to add documentation.

Reply:
> 1. `level` - uses the CII level for the badge message with a simple color scheme
> 2. `percentage` uses the CII score/percentage with the standard Shields percentage color scheme
> 3. `default` - uses the same badge message & color scale as the native CII badge provides, but with all the configuration and goodness that Shields provides! (I'm not crazy about `default` as the path name/label but I wasn't sure what else to call this one)

`status` or `summary` might be a more descriptive name that emphasizes the content that’s being shown.

Reply:
Yeah I was having similar thoughts myself, but just wasn't sure what to put in there for the documentation. I like the idea though!

@paulmelnikow - are you suggesting `status` or `summary` as a replacement for `default` (both of which are far better than `default`) and/or in place of the other two (`level` or `percentage`)

Reply:
i changed `default` to `summary`